bin/pint

```diff
@@ -150,22 +151,94 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        info = False
+        no_info = ''
+        data_json = {}
+        data_xml = None
+        for reg in region.split(','):
+            data = ifsrequest.get_image_data(
+                framework,
+                image_state,
+                output_format,
+                reg.strip(),
+                command_args['--filter']
+            )
+            if '<images/>' in data or '"images": []' in data:
+                info = False
+                continue
+
+            info = True
+            # there is info in some region/s
+            # so we show the empty response/s
+            # and not the message
+            if output_format in output_options:
+                if 'json' in output_format:
+                    data = json.loads(data)
+                    data_json['images'] = data_json.get('images', []) + data.get('images')
+                    data = data_json
```

Comment:
data get over written in the next loop `data = ifsrequest.get_image_data( ...` on line 159 this assignment is superflous.

Also if `data_json` is properly initialized as `data_json{'images': []'}` the expression simplifies to `data_json['images'] += data.get('images')`

Reply:
if `data` or any other common variable for both cases (`json` and `xml`) when returning/printing the info after the loop,
there would have to be another check to print the output properly (json.dumps or etree.tostring, relatively), which I guess check again is no biggie